src/content/tutorials/fleet-management/cluster-management/tunning-cluster-upgrades/index.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Avoid disruption of workloads during cluster upgrade
+description: The level of disruption caused by cluster upgrades can be influenced per cluster. This article explains how to adjust the number of nodes that is updated simlutaneously, and the wait time between batches of nodes.
+weight: 10
+menu:
+  principal:
+    parent: tutorials-fleet-management-clusters
+    identifier: tutorials-fleet-management-tunning-cluster-upgrades
+user_questions:
+  - How can I limit the disruption caused by cluster upgrades?
+last_review_date: 2024-10-07
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+---
+
+## Introduction
+
+Cluster upgraded, described in detail in our [cluster upgrades reference]({{< relref "/vintage/platform-overview/cluster-management/cluster-upgrades" >}})), can cause disruption on workloads, if the upgrade requires upgrading worker nodes.
+
+We provide two ways of limiting the amount of disruption:
```

Comment:
From here, this is all outdated and not implemented for CAPI. `AWSMachinePool` takes care of the instance refresh, and its settings dictate what happens. We'd first need to work on making this more stable (e.g. https://github.com/giantswarm/giantswarm/issues/31843). Or if you want, we can describe the current behavior already and update the article again later. Mind however that it will be provider-specific, since for CAPA we use machine pools, for others we don't.

Reply:
It is nice to have, but we can wait if it changes soon. When new process is implemented we can work here to make it up to date